File: tests/python/programs/fizzbuzz.py
# vybe-test: python/programs/fizzbuzz
# origin: languages/python/tests/python/test_programs.rs
# vybe-test-mode: compile

for i in range(1, 101):
    if i % 15 == 0:
        print("FizzBuzz")
    elif i % 3 == 0:
        print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print(i)

File: tests/fortran/programs/fizzbuzz.f90
! vybe-test: fortran/programs/fizzbuzz
! origin: languages/fortran/tests/fortran/test_programs.rs

program fizzbuzz
    integer :: i
    do i = 1, 15
        if (mod(i, 15) == 0) then
            print *, "FizzBuzz"
        else if (mod(i, 3) == 0) then
            print *, "Fizz"
        else if (mod(i, 5) == 0) then
            print *, "Buzz"
        else
            print *, i
        end if
    end do
end program fizzbuzz
